src/routes/sign-routes.ts

```diff
@@ -0,0 +1,40 @@
+import { Response, Request, Application } from "express";
+import Signature from "../signature";
+import models from "../models";
+
+/**
+ * Sign routes
+ */
+export default class SignRoutes {
```

Comment:
If this code is moved from the "src-old" it should be removed from it when migrated into new code